src/test/java/uk/gov/hmcts/darts/common/util/CommonTestDataUtil.java

```diff
@@ -160,10 +161,18 @@ public HearingEntity createHearing(String caseNumber, LocalTime time) {
         hearing1.setHearingDate(LocalDate.of(2023, 6, 20));
         hearing1.setScheduledStartTime(time);
         hearing1.setId(102);
+        hearing1.setTranscriptions(createTranscription());
         hearing1.addJudges(createJudges(2));
         return hearing1;
     }
 
+    public List<TranscriptionEntity> createTranscription() {
+        TranscriptionEntity transcription = new TranscriptionEntity();
+        transcription.setCompany("Transcription company");
+        transcription.setId(1);
```

Comment:
I was meaning the `trt_id` - what we have called `transcriptionTypeId` in our entity
Seeing as you have added an insert sql script for them also?

Reply:
done